### ISSUE TITLE
Watermark screens " Put your Logo Here " 

### ISSUE THREAD

**Issue by synchromesh62** [TheGameCreators/GameGuruRepo#659]
Regarding the Watermark screens contained in In Languagebank \ English \ artwork. 
Any chance this could be disabled or removed as it just another 12 screens we have to edit on top of the already 40 screens we need to edit \ resize for our menu's.
They really don't serve any purpose and are just extra work for us to get the professional look :(

**Comment by Kasseyus** [comment]
enhancement is the closest label there is.  'New Feature of Request'.  It's not a bug, so to speak, but that label has been added also, as it involves the removal of unwanted code.

I`m sure it will be addressed as more of a tweak however, as it is removing something rather than adding something which is always going to be less work.  (one would hope!)

**Comment by synchromesh62** [comment]
Ok thanks Kasseyus.. The ironic thing is last year they didn't work and were never used .. Someone fixed the bloody things … Typical :)

**Comment by michaeloldman** [comment]
Yeah, all screens in standalone should be sourced only from the TitlesBank folder.

@Kasseyus 
The ones in language bank serve no useful purpose - period.
Since we use **style txt to tell GG what to use for screens in standalone**, which in turn **uses the corresponding folders within titlesbank**, pulling them from anywhere else is essentially a bug.

Definitely **not an enhancement** so please loose that label.
It is most certainly a **bug** that needs squashing.

**Comment by synchromesh62** [comment]
@LeeBamberTGC 
@plemsoft 
Not sure which this is really classed as

**Comment by Teabone3** [comment]
I use the watermark screen to put my own logo before the main menu title screen loads up.
Perhaps an option to skip them in setup.ini or elsewhere. would help for those that dont use them.
